build/Localization/LocalizationAutomation.proj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\Microsoft.WDG.Localization.Touchdown.Build.Client.1.0.5\build\Microsoft.WDG.Localization.Touchdown.Build.Client.props" Condition="Exists('..\..\packages\Microsoft.WDG.Localization.Touchdown.Build.Client.1.0.5\build\Microsoft.WDG.Localization.Touchdown.Build.Client.props')" />
+  <Import Project="..\..\packages\Microsoft.WDG.Localization.Touchdown.Build.Client.1.0.13\build\Microsoft.WDG.Localization.Touchdown.Build.Client.props" Condition="Exists('..\..\packages\Microsoft.WDG.Localization.Touchdown.Build.Client.1.0.13\build\Microsoft.WDG.Localization.Touchdown.Build.Client.props')" />
   <PropertyGroup>
```

Comment:
Are the localization changes intentional ?

Reply:
No.  The original changes were just for the headers and sources.  It added a bunch of files on the merge (which I needed to do to pull the changes for RuntimeProfiler.h).

I could abandon, apply the changes to current master, and resubmit, if it'll make it easier to read through.

Reply:
what you see here is what will get merged as your change. You should remove those changes either manually or by reapplying your changes on latest master.